enhancements/monitoring/monitoring-stack-operator.md

```diff
@@ -0,0 +1,490 @@
+
+The following diagram illustrates a high-level overview of the final solution applied in the context of managed services.
+
+![drawing](https://docs.google.com/drawings/d/e/2PACX-1vT4p10FDLEg0bv_pLUlwyVszq1275MNRJ-sBZMLxT5gcZ0q-tY8MHoEYpOlVmJuxlT40lqUQYCDwaSC/pub?w=1213&h=898)
+
+_First iteration of the new monitoring operator, showing a monitoring stack with a Prometheus, Grafana and a shared Alertmanager._
+
+We recommend using either [OperatorSDK](https://github.com/operator-framework/operator-sdk) or [KubeBuilder](https://github.com/kubernetes-sigs/kubebuilder) to develop the operator to increase development efficiency by taking advantage of the higher abstraction constructs, the rich documentation, and a vibrant community.
+Alternatively, we could also consider using a higher-level operator, such as [locutus](https://github.com/brancz/locutus), for rapid development with a path for GitOps deployments. In any case, this is an implementation detail that should be left to the team implementing this to decide on.
+
+## Design Details
+
+### Installation
+
+The newly developed operator will be deployed using the Operator Lifecycle Manager (OLM).
+
+This will allow us to make releases much faster and iteratively and incrementally. As a result, we will be able to respond to new requirements much more rapidly. Release batches will also become smaller, reducing the risk of multiple independent features affecting each other.
+
+In order to be aligned with the descoping [plan](https://docs.google.com/presentation/d/1j1J575SxS8LtL_YvKqrexUhso7j4SgrLfyNrDUroJcc/edit#slide=id.ga089527607_0_0) that the OLM team is currently working towards, we recommend avoiding `OperatorGroups` as they are going to become deprecated in the future (possibly as soon as OpenShift 4.11).
+For this reason, the newly developed operator should be installed globally in a cluster and should provide its own CRDs and Roles as part of the installation bundle. A cluster administrator would then create RoleBindings to bind the roles to namespaces to which the operator needs access.
+More details about the descoping plans and strategies are currently described can be found in this [hackmd](https://hackmd.io/wVfLKpxtSN-P0n07Kx4J8Q?view#Descoping-Plan) document and are also illustrated in a short [slide-deck](https://docs.google.com/presentation/d/1j1J575SxS8LtL_YvKqrexUhso7j4SgrLfyNrDUroJcc/edit#slide=id.ga089527607_0_0).
+
+### Metrics retention
+
+Since each addon can create its own monitoring stack, MTOs will be able to configure retention for their service as they see fit.
+
+### Global view
+
+In order for MTSRE/MTO to be able to view portions of metrics for all of their deployments, each managed service will have its own tenant in Red Hat Observability (RHOBS). This will allow remote-writing a subset of metrics from the Prometheus instances into RHOBS.
```

Comment:
Do we plan to track the changes needed into RHOBS separately? Is there a design doc for how this tenancy would look like?

Reply:
I guess [Observatorium](https://github.com/observatorium/observatorium) could be a more apt reference, since Observatorium (more precisely the [API](https://github.com/observatorium/api)) is what enables these features already at the present (although I'm unable to find any further design doc at this moment, regarding specifically the tenancy feature). RHOBS is then our hosted instance of Observatorium. 

Maybe expanding on this / adding few words / adding references on RHOBS <-> Observatorium here or elsewhere in the document could be beneficial.

Reply:
nit 
```suggestion
In order for MTSRE/MTO to be able to view portions of metrics for all of their deployments, each managed service will have its own tenant in Red Hat Observability Service (RHOBS). This will allow remote-writing a subset of metrics from the Prometheus instances into RHOBS.
```